src/test/java/uk/gov/hmcts/reform/civil/sampledata/CaseDataBuilder.java

```diff
@@ -225,15 +230,15 @@ public CaseDataBuilder respondent2OrganisationPolicy(OrganisationPolicy responde
         return this;
     }
 
+    public static CaseDataBuilder builder() {
+        return new CaseDataBuilder();
+    }
+
     public CaseDataBuilder atStateClaimDraft() {
 
         return this;
     }
 
-    public static CaseDataBuilder builder() {
```

Comment:
why are we removing this code block?

Reply:
Removed it accidently but added it on lines 233 - 236